crates/streamer/src/streamer/iter.rs

```diff
@@ -3,7 +3,7 @@ use std::io::Seek;
 use anyhow::Context;
 use cadeau::xmf::vpx::is_key_frame;
 use thiserror::Error;
-use tracing::debug;
+use tracing::{debug, trace};
```

Comment:
Did you use `#[macro_use]` above `extern crate tracing` in lib.rs?

Reply:
I can do this in separate PR